src/main/core/support/configuration.rs

```diff
@@ -1,8 +1,9 @@
-use std::collections::BTreeMap;
```

Comment:
It'd be nice if we caught somewhere if any of the specified host names aren't host names in the simulation. Maybe we can do it somewhere here? (Also fine to consider that as something to add later)

Reply:
Yeah, I think it would be good but I'll leave it as something to do later. It would be much easier to do when the worker is in rust, since we need to wait until the hostnames are expanded before we check. For example, a host named 'client' in the config may get renamed to 'client1' and 'client2' if `quantity` is 2.